examples/tv-app/linux/include/application-basic/ApplicationBasicManager.cpp

```diff
@@ -0,0 +1,168 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "ApplicationBasicManager.h"
+
+#include "gen/attribute-id.h"
+#include "gen/attribute-type.h"
+#include "gen/cluster-id.h"
+#include "gen/command-id.h"
+
+#include <app/util/af.h>
+#include <app/util/basic-types.h>
+
+#include <inipp/inipp.h>
+
+#include <fstream>
+#include <iostream>
+#include <sstream>
+using namespace std;
+
+CHIP_ERROR ApplicationBasicManager::Init()
+{
+    CHIP_ERROR err                                       = CHIP_NO_ERROR;
+    EndpointConfigurationStorage & endpointConfiguration = EndpointConfigurationStorage::GetInstance();
+    err                                                  = endpointConfiguration.Init();
+    SuccessOrExit(err);
+    es = &endpointConfiguration;
+exit:
+    return err;
+}
+
+void ApplicationBasicManager::store(chip::EndpointId endpoint, Application * application)
+{
+    if (strcmp(application->vendorName, ""))
```

Comment:
What's the reasoning behind making all these attribute writes conditional?

Reply:
To have the log if storing failed.

Reply:
Aha, you've meant for the conditional which verified if value is correct. Yeah that one is not necessary I've removed it.